perception/tensorrt_bevdet/package.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0"?>
+<package format="3">
+  <name>tensorrt_bevdet</name>
```

Comment:
I would recommend to add `autoware_` prefix, as you would need to do it anyway.
https://github.com/autowarefoundation/autoware/issues/4569

Reply:
> I would recommend to add `autoware_` prefix, as you would need to do it anyway. [autowarefoundation/autoware#4569](https://github.com/autowarefoundation/autoware/issues/4569)

In the current `autoware.universe` repository, only `autoware_` is used  as a prefix for the packages name of autoware's own utils or plugins. This method is not used for naming other packages. In order to be consistent with the naming rules of other packages, I used this name as the package name.

Reply:
@cyn-liu Thank you for the feedback. But changing the package name afterwards is a bit tricky task (which may involve modification not only within this package but also in others as well), so let me double check that the `autoware_` prefix stuff can have exceptions like this or not. 

@esteve Hi, let me mention you as I assume you are one of the lead developer in introducing `autoware_` prefix. Is current initiative applicable to all the packages within `awf/autoware.universe` ? Or can it have a certain exception like this `tensorrt_bevdet`?